google-cloud-bigtable/test/google/cloud/bigtable/project/table_test.rb

```diff
@@ -72,34 +72,6 @@ def stub.get_table *args
     _(table).must_be :nil?
   end
 
-  it "load schema view on access schema fields" do
```

Comment:
looks like this was migrated to the other `table_test.rb file`, is that correct? (forgive the simple question, what's the difference?)

Reply:
Yes, since this test did not involve `Project` in any way, I moved it to a more appropriately scoped file.